src/ProjectTemplates/Web.ProjectTemplates/Components-CSharp.csproj.in

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <!--#if UseClient -->
```

Comment:
Would it work to use the terminology WebAssembly rather than Client? There are client-side pieces whether or not there is WebAssembly.